scripts/zones/Lower_Jeuno/npcs/Treasure_Coffer.lua

```diff
@@ -532,6 +1619,7 @@ local argumentKeyItems =
 }
 
 entity.onTrigger = function(player, npc)
+    local eligableNexusCape = (xi.settings.main.ENABLE_ACP * xi.settings.main.ENABLE_AMK * xi.settings.main.ENABLE_ASA) ~= 0
     local receivedNexusCape = player:getCharVar('receivedNexusCape')
```

Comment:
The logic in here, is this a "you've recieved this item once and then thrown it away, so you're boned" scenario? or could this be changed to just look in a player's inventory?

Or is this one of those scenarios I'm a bit afraid of where it's an item _per-account_ and we'd have to look up whether or not any mules have it, or it's in dbox, etc.?

Reply:
yea the box has 2 different menu options, one for getting it the first time and one for getting it an additional time

Reply:
also `eligableNexusCape` is determining if nexus cape should be obtainable at all. retail is "purchased all 3 mini expansions", i'll be making a module to adjust this bool to be "completed all 3 mini expansion"

Reply:
added comments to make it more explicit what is going on